tests/e2e/pageobjects/ide/OpenWorkspaceWidget.ts

```diff
@@ -31,9 +31,14 @@ export class OpenWorkspaceWidget {
         const pathNodes: string[] = path.split('/');
         let buildPath: string = '';
 
-        const promises = pathNodes.map(async currentPath => {
+        const promises = pathNodes.map(async (currentPath, index) => {
             buildPath += `/${currentPath}`;
-            await  this.driverHelper.waitAndClick(By.id(buildPath), timeout);
+
+            if (index === 0) {
```

Comment:
This if/else seems a little bit magic to me, maybe small comment would ease to understand :)

Reply:
Yes, already prvided